internal/plan/plan.go

```diff
@@ -27,33 +27,36 @@ import (
 	"gopkg.in/yaml.v3"
 )
 
-type Setup struct {
-	Layers []*Layer
+type Plan struct {
+	Layers      []*Layer            `yaml:"-"`
+	Summary     string              `yaml:"summary,omitempty"`
+	Description string              `yaml:"description,omitempty"`
```

Comment:
Per conversation online, we need to decide what to do with these before we have fields here.

Reply:
Fixed. Plan no longer has summary/description.